hs_err_pid47193.log

```diff
@@ -0,0 +1,612 @@
+#
```

Comment:
I'm assuming committing this was unintentional ;)

Probably worth updating git ignore...

Reply:
ack